app/client/components/identity/idapi/user.ts

```diff
@@ -32,6 +32,15 @@ type UserPrivateFields = Partial<
 	};
 };
 
+// The api error message is displayed directly to the user unless
+// you create an MMA specific error message here per field.
+const userErrorMessageMap = new Map([
+	[
+		'user.privateFields.registrationLocation',
+		'Please select a location from the list or "Prefer not to say"',
```

Comment:
This isn't really necessary anymore given the "unknown" option has been disabled, however,  if this error is returned by the api for for another reason (?), it's probably nice to have the bespoke message